itsm_api/views.py

```diff
@@ -1447,13 +1447,16 @@ def _get(self, comment_id):
             flask_restful.abort(404, message=f"Comment '{comment_id}' not found!")
         ticket_data   = DB_TICKET_TABLE.get(doc_id=comment['ticket_id'])
         customer_data = DB_CUSTOMER_TABLE.get(doc_id=ticket_data['customer_id'])
-        user_data     = DB_USER_TABLE.get(doc_id=comment['user_id'])
+        user_data = {}
+        if comment.get('user_id'):
```

Comment:
Shouldn't we also attempt to retrieve the user data via the email, if it was provided instead of the user ID?

Reply:
Since you said it will be taken care in the wrapper, i didnt do any steps for that